### DISCUSSION TITLE
Replacement for FrontstageManager.setActiveContentGroup()

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#5593] [Q&A]
It seems ```FrontstageManager.setActiveContentGroup()``` is deprecated in 3.7 and there is no replacement in ```UiFramework.frontstages```.  Can you please guide what do I need to use.

I looked at samples and they are still using ```FrontstageMananger``` and documentation for ```setActiveContentGroup()``` is no longer available.  Usually old documentation has info on what to use but in this case I get a 404 error.  I couldn't find any info in release notes in breaking changes section either.

I am still using 3.7 and it seems I am unable to access 3.7 documentation anymore.  Can you please guide how I can access 3.7 documentation.  Changing to 3.x in drop down doesn't see to take me to old documentation,

Thanks,

Ovais

**Comment by pmconne** [comment]
@ben-polinsky @NancyMcCallB @raplemie

**Comment by raplemie** [comment]
Hi Ovais, 
   you are right that the moved individual methods were not properly documented in code, here is an exaustive list of the different manager classes that were changed: https://github.com/iTwin/itwinjs-core/blob/release/3.7.0/docs/changehistory/3.7.0.md#static-manager-classes
   
   In this list, you will see that `FronstageManager.setActiveContentGroup()` is now accessed through `UiFramework.content.layouts.setActiveContentGroup`.

@ben-polinsky will be able to guide you to reach the proper 3.x documentation.

Raphael

**Comment by ben-polinsky** [comment]
@ovaisa Apologies for any inconvenience. The 3.x docs should be usable now.

Edit: to clarify. Clicking on the v3 link in the dropdown will now properly scope all links. Looks like we do have some issues with search scoping results to v3, however. Looking into that now.

**Reply by ben-polinsky** [reply]
And search is now working on v3.

**Reply by ovaisa** [reply]
I am afraid v3 drop down is still not working for me, taking me to https://www.itwinjs.org/v3/.  V2 works OK.  Thought I am able to access V3 documentation at https://www.itwinjs.org/v3/reference/
